test/unit/points/test-calculate-community-workload-points.js

```diff
@@ -0,0 +1,14 @@
+const expect = require('chai').expect
+const calculateCommunityWorkloadPoints = require('../../../app/points/calculate-community-workload-points')
+const pointsHelper = require('../../helpers/points-helper')
+const Locations = require('../../../app/staging/constants/locations')
+
+describe('points/calculate-community-workload-points', function () {
+  it('should calculate the value without error', function () {
+    var tiersObject = pointsHelper.getTestTiersObject(Locations.COMMUNITY)
+    var caseTypeWeightings = pointsHelper.getWeightings()
+
+    var result = calculateCommunityWorkloadPoints(tiersObject, caseTypeWeightings, true, true)
```

Comment:
I think I discussed this with @andrew-js-wright in that we can asssume that they will always be populated. There could be another scenario for the other two switches being off. Which I will add in the next commit.